cmd/oras/internal/input/file_windows.go

```diff
@@ -0,0 +1,25 @@
+package input
```

Comment:
Can we have a better name? Also, the license header is missing.

Reply:
Added license. Renamed to `parse`